src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    private static final int TITLE_INDEX = 0;
+
+    @Override
+    public List<String> readFile(String filePath) {
+        List<String> fruitTransaction;
+        try {
+            fruitTransaction = Files.readAllLines(Path.of(filePath));
+            fruitTransaction.remove(TITLE_INDEX);
```

Comment:
This class should only read data from the file without any modification, so le't move this logic to another place
```suggestion
            fruitTransaction.remove(TITLE_INDEX);
```

Reply:
fix comment